libs/core/mappers/ordering.mapper.ts

```diff
@@ -0,0 +1,36 @@
+/** Ordering. */
+export interface Ordering<Field, Direction> {
+
+	/** Field. */
+	readonly field: Field;
+
+	/** Direction. */
+	readonly direction: Direction;
+}
+
+/** Ordering mapper for field or direction. */
+type OrderingMapper = Record<string, string>;
+
+/** Ordering mapper. */
+export namespace OrderingMapper {
+
+	/**
+	 * Converts ordering model to DTO.
+	 * @param ordering Model.
+	 * @param orderingFieldMapper Converts field model to DTO.
+	 * @param orderingDirectionMapper Coverts direction model to DTO.
+	 */
+	export function toDto<Field extends string, Direction extends string>(
+		ordering: Ordering<Field, Direction> | undefined,
+		orderingFieldMapper: OrderingMapper,
+		orderingDirectionMapper: OrderingMapper,
+	): string | undefined {
+		if (!ordering) {
```

Comment:
Please dont use implicit typecasting:
https://wiki.saritasa.rocks/frontend/languages/ts-js/best-practices#dont-use-implicit-type-casting

```suggestion
		if (ordering == null) {
```